test/spec/todo-item/todo-item.spec.ts

```diff
@@ -87,7 +87,44 @@ describe('POST /todos', () => {
       expect(res).to.have.status(400)
       expect(res.body)
         .to.have.nested.property('failures[0].message')
-        .to.equal('The title is empty or the title is not a string.')
+        .to.equal(
+          'The title name is already taken. Please enter a different one.'
+        )
     }
   })
+
+  describe('DELETE /todos/:id', () => {
+    it('should delete a todo item if it exists and if ID is valid MongoDB ID', async () => {
+      const todoItem = await testAppContext.todoItemRepository.save(
+        new TodoItem({ title: 'Title to be deleted' })
+      )
+      const res = await chai
+        .request(expressApp)
+        .delete(`/todos/${todoItem._id}`)
+
+      expect(res).to.have.status(204)
+    })
+
+    it('should return a validation error if todo item does not exists and if id is a mongo id', async () => {
+      const res = await chai
+        .request(expressApp)
+        .delete('/todos/60d2fe74bd99a211407165e9')
+
+      expect(res).to.have.status(400)
+      expect(res.body)
+        .to.have.nested.property('failures[0].message')
+        .to.equal(
+          'The item with the specified ID could not be found. Please enter a valid ID.'
+        )
+    })
+
+    it('should return a validation error if id is not a mongo id', async () => {
+      const res = await chai.request(expressApp).delete('/todos/2114071')
+
+      expect(res).to.have.status(400)
+      expect(res.body)
+        .to.have.nested.property('failures[0].message')
+        .to.equal('The specified ID is not a MongoDB ID.')
```

Comment:
Nitpick - Generally internal workings of the application are not exposed to public and user messages are platform agnostic. Meaning that `The specified ID is not a MongoDB ID` could have been better rewritten as `The specified todo ID is not a valid one. Please provide a valid one to continue.`. This would work irrespective of the underlying database layer as well as does not exposes internal structure of the app.